### PR TITLE
build: add deploy workflow

### PR DESCRIPTION
**Pull request by jordan-a-young**
This PR adds some docs, scripts, and the github action workflow for publishing a release

### CODE REVIEW

File: .github/workflows/pr-build.yml [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 
 jobs:
   # This job will:
-  #   * deploy a draft every time there is a pull request created or synchronized that is not on main branch
-  #   * comment on that pull request with the deploy URL
+  #   * build each package
+  #   * run unit tests
+  #   * run the linter
   build:
-    if: ${{ github.actor != 'dependabot[bot]' }}
```
Comment (LauRoxx) [inline .github/workflows/pr-build.yml:15]: have to take out the token for the checkout code step for dependabot to run.

File: packages/design-tokens/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@availity/design-tokens",
-  "version": "0.5.0",
+  "version": "0.4.0",
```
Comment (LauRoxx) [inline packages/design-tokens/package.json:3]: I'm assuming you meant for the version number to decrease but calling attention jic.

File: .github/workflows/pr-build.yml (hunk 1 above)
Comment (jordan-a-young) [reply .github/workflows/pr-build.yml:15]: I removed it. Thanks for the heads up

File: packages/design-tokens/package.json (hunk 2 above)
Comment (jordan-a-young) [reply packages/design-tokens/package.json]: Yup! I want the version to be bumped to 0.5.0 so have to start at 0.4.0

**Review by LauRoxx — APPROVED** [review]

**Pull request merged**